rancher/v1.6/en/cattle/volumes/index.md

```diff
@@ -0,0 +1,150 @@
+
+<sup>1</sup> Cross-host volume sharing is subject to AWS `Availability Zone` and `Region` limitations. At the time of writing, Rancher scheduler enforces these rules for EBS volumes.
+
+> **Note:** Restarting Docker daemon sometimes incurs a large period of latency when containerized volume plugins are in use. This is unfortunately out of our control. A prioritization scheme needs to be implemented in Docker to restart plugin containers prior to plugin-dependent containers.
+
+### Example Usage
+
+#### V1 Compose
+
+> **Note:** All volumes created with V1 compose are environment scoped. 
+
+Below is a V1 `docker-compose.yml` file for creating a container with a volume named `etcd` that implicitly uses the `local` volume driver.
+
+```yaml
+etcd:
+    image: rancher/etcd:v2.3.7-11
+    volumes:
+    - etcd:/pdata
+```
+
+In the next example V1 `docker-compose.yml`, we create a container with a volume named `etcd-backup` that uses the `rancher-nfs` volume plugin. It is expected that the plugin is already installed and configured in the target environment.
+
+```yaml
+etcd:
+    image: rancher/etcd:v2.3.7-11
+    environment:
+        EMBEDDED_BACKUPS: 'true'
+        BACKUP_PERIOD: 5s
+        BACKUP_RETENTION: 15s
+    volumes:
+    - etcd_backup:/data-backup
+    volume_driver: rancher-nfs
+```
+
+> **Note:** Docker V1 Compose is limited to using a single volume driver for all named volumes. This limitation is addressed in V2 compose.
+
+See the [V1 Compose Reference](https://docs.docker.com/compose/compose-file/compose-file-v1/#volumes-volume_driver) for more information.
+
+#### V2 Compose
+
+The next example is equivalent to the last, except we will also persist the data directory of each etcd container to its own local volume (container scoped). The `external: true` modifier tells Rancher that the volume is environment scope. Rancher treats V2 environment scope volumes different from V1; a user is responsible for creating the external volume beforehand. Without any modifier, Rancher interprets V2 volumes as stack scoped and will manage their lifecycle.
+
+Any external volume may be created via Rancher CLI; volumes that use a driver besides `local` may be created from Rancher UI `infrastructure >> storage` tab.
+
+We will use Rancher CLI. First, create the external volume via `rancher volume create --driver rancher-nfs etcd_backup`. Create the stack via `rancher stack create` where the current working directory contains the `docker-compose.yml` below. Just for run, scale the service up to 3.
+
+```yaml
+version: '2'
+services:
+    etcd:
+        image: rancher/etcd:v2.3.7-11
+        environment:
+            EMBEDDED_BACKUPS: 'true'
+            BACKUP_PERIOD: 1s
+            BACKUP_RETENTION: 10s
+        volumes:
+        - etcd_backup:/data-backup
+volumes:
+    etcd_backup:
+        driver: rancher-nfs
+        external: true
+```
+
+> **Note:** V2 volumes defined with the `local` volume driver do not work as of Rancher `1.6.2`
```

Comment:
What doesn't work about them? The `driver: "null"` bug? That will be fixed in 1.6.3. I don't think we should mention that here; it's confusing.